src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const object = {
+    result: 0,
+  };
+
+  object.add = function (number) {
+    object.result += number;
+  };
```

Comment:
Instead of using the 'function' keyword, you can use the method shorthand to define your methods. This will make your code more concise and modern.